src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -855,6 +855,9 @@ boolean getServerSupportsColumnEncryption() {
     ColumnEncryptionVersion getServerColumnEncryptionVersion() {
         return serverColumnEncryptionVersion;
     }
+    
+    /** whether the server supports retrying a connection on failure */
+    private boolean serverSupportsRetry = false;
```

Comment:
Minor NIT: The variable name should be more specific. Maybe serverSupportsEnclaveRetry?